### ISSUE TITLE
Warning alert each time updating the web client version or on re-build.

### ISSUE THREAD

**Issue by maxsdw** [RocketChat/Rocket.Chat.Cordova#105]
Every time the web client version is updated or on re-builds, this alert starts appearing:

```
Warning

The URL configured in your server
(https://...) is not
the same that you are using here
```

Right now the only solution is re-installing the app in the device.
##

**Comment by engelgabriel** [comment]
iOS or Android?

**Comment by maxsdw** [comment]
Both.

**Comment by engelgabriel** [comment]
About when you access our demo server?

**Comment by rebeld** [comment]
I've seen something similar with my users if they don't explicitly use '**https://**ourchatserver.com' as their url when they configure their mobile apps (i.e: they just use ourchatserver.com, or '**http://**ourchatserver.com'). They also tend to see some weird behaviour (i.e: the user tagging list when typing '@something' doesn't render correctly) Deleting the server and recreating seems to remove the nag and resolves their issue.

**Comment by geekgonecrazy** [comment]
@rebeld by delete server you mean just removing it on the mobile app right?  Just want to make sure you aren't having to re-install the server :smile:

**Comment by engelgabriel** [comment]
Delete SERVER form the SERVER LIST on the mobile :)

**Comment by rebeld** [comment]
Yes, what @engelgabriel said. Definitely not delete our server install ;)

**Comment by geekgonecrazy** [comment]
@rebeld I thought so.  I just wanted to make sure!  😆

**Comment by maxsdw** [comment]
Gabriel, We are accessing our own server where we have an instance of RC.

**Comment by geekgonecrazy** [comment]
@maxsdw but when you connect are you specifying exactly https://yoursite ?  or http://yoursite ?

**Comment by maxsdw** [comment]
yes, in all cases, we always use **https://...**

**Comment by geekgonecrazy** [comment]
@maxsdw ok just wanted to make sure as that was the resolution for some.  I thought I had asked you when we talked a few days ago.  But I couldn't remember :smile:

**Comment by maxsdw** [comment]
Do you know what part of the code is generating that alert? so maybe I can just comment-out that part in order to prevent that alert appearing.

**Comment by bennlich** [comment]
Looks like it's happening here:

https://github.com/RocketChat/Rocket.Chat.Cordova/blob/649f7e6545df7e196ff807c420ead800008eab88/www/coffee/servers-list.coffee#L135

This issue seems to be related (or a dupe) of https://github.com/RocketChat/Rocket.Chat.Cordova/issues/34.
